spec/spec_helper.rb

```diff
@@ -1,8 +1,138 @@
-require 'coveralls'
+# Generated by rspec-sinatra. (2015-07-03 11:26:50 +0100)
+
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'app/chitter.rb')
+
+require 'capybara'
 require 'simplecov'
+require 'coveralls'
+require 'capybara/rspec'
+require 'rspec'
+require 'factory_girl'
+require './app/data_mapper_setup'
+require 'database_cleaner'
+require './spec/factories/user'
+require_relative './helpers/helper'
+
+Capybara.app = CHITTERchallenge
+
 
 SimpleCov.formatters = [
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
+ SimpleCov::Formatter::HTMLFormatter,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.